calendar/Package.md

```diff
@@ -0,0 +1,20 @@
+Connects to Microsoft Outlook Calendar from Ballerina
+
+### Package Overview
+The `outlook.calendar` is a [Ballerina](https://ballerina.io/) connector for Outlook Calendar.
+This package provides the capability to easily access Outlook Calendar.
+
```

Comment:
Somewhere we need to point to module.md file. In overview, we do not provide much detail, but we need to point where this info is. @Chuhaa where should that come?

Reply:
In the package.md, we don't provide it manually as central has the API document link in the page.

Reply:
Yeah. Let's go with that approach then.